_posts/2024-05-14-switch-to-bam.md

```diff
@@ -0,0 +1,181 @@
+---
+layout: post
+title: What does it take to switch over to bam from fastq?
+tags:
+  - bam
+  - fastq 
+contributors:
+  - lskatz
+  - SeanSierra-Patev
+reviewers:
+  - pmenzel
+  - hexylena 
+---
+
+It has been over 14 years since the formalization of the fastq format ({% cite Cock2009 %}),
+which describes sequences and their quality scores.
+For paired end reads, sequences are encoded in separate files, usually called R1 and R2 [^1].
+Unfortunately despite the publication,
+fastq format is not entirely standardized!
+For example, it is possible to have a valid fastq format in either 4-line-per-entry format,
+or splitting sequences into multiple lines.
+Additionally, the defline itself is not entirely standardized and is basically free text.
+However, we have had a lot of innovations in sequence formats since then.
+One of those innovations is the BAM format which is the binary alignment/mapping format.
+R1 and R2 are encoded in the same file.
+
+Fastq files are integral to genomic epidemiology.
+State health labs sequence genomes,
+transfer the fastq files to an internal repository,
+run quality checks (QC), usually through a quality assurance (QA) pipeline.
+
+**So what would it take to change this whole process to bam files instead?**
+Bam files have many advantages including having one file per sample,
+encoding extra information such as alignment data,
+and being indexed for random access.
+For our purposes here, bam files will be unaligned bam (uBAM)
+because they are not aligned against anything.
+
+## Sequencing
+
+First, the sequencers would have to output uBAM files.
+Can they?
+The Illumina platforms and the Ion Torrent platforms do automatically.
```

Comment:
I didn't see Illumina being able to output BAM just for the sequencing data.

Reply:
Thanks. I'll keep looking just in case. I don't want to get that fact wrong in particular.

Reply:
I had added in Illumina text per the representative